lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -88,6 +88,22 @@ public static function find(array $filters = []) {
   private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
+
+    $northstar_response = dosomething_northstar_get_northstar_user($data->uid);
```

Comment:
That's only if the full user is needed, but all we need here is the ID, correct?